src/lib/rule-finder.js

```diff
@@ -31,9 +31,16 @@ async function _getConfigs(overrideConfigFile, files) {
   return new Set((await Promise.all(configs)).filter(Boolean));
 }
 
+function _getInputArrayOrEmpty(arr) {
+  return arr ? arr : []
+}
```

Comment:
first, this should be `arr || []`, and second, you want `[].concat(arr || [])`, which can just be inlined.
```suggestion
```